monai/handlers/tensorboard_handlers.py

```diff
@@ -173,6 +173,21 @@ def iteration_completed(self, engine: Engine) -> None:
         else:
             self._default_iteration_writer(engine, self._writer)
 
+    def _write_scalar(self, engine: Engine, writer: SummaryWriter, tag: str, value: Any, step: int) -> None:
+        """
+        Write scale value into TensorBoard.
+        Default to call `SummaryWriter.add_scalar()`, subclass can override it for more complicated logic.
```

Comment:
I think there's no need to mention the override, because it's a python buildin feature. Also perhaps we start variable names with an underscore for unused variable

Reply:
OK, updated in the latest commit, and also changed unused arg in all the handlers.

Thanks.

Reply:
I feel if it is unused, there's no need to mention in the docstring that it can be trainer/evaluator...could you help update them as well?

Reply:
Sure, changed to unused argument in the latest commit.

Thanks.